### PR TITLE
Add notification center API

### PR DESCRIPTION
**Pull request by assafad1**

### CODE REVIEW

File: go/connectors-client.go [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+type GetConnectorResponse = connectores.GetConnectorResponse
+
+// ListConnectorsRequest represents a request to list connectors.
+type ListConnectorsRequest = connectores.ListConnectorsRequest
+
+// ListConnectorsResponse represents a response to list connectors.
+type ListConnectorsResponse = connectores.ListConnectorsResponse
+
+// BatchGetConnectorsRequest represents a request to batch get connectors.
+type BatchGetConnectorsRequest = connectores.BatchGetConnectorsRequest
```
Comment (celaus) [inline go/connectors-client.go]: for those bulk/batch requests, we already have some APIs exposing something similar, can you make sure they all use the same noun to describe it? so it's either batch or bulk across the SDK

File: go/connectors-client.go [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+// BatchGetConnectorsResponse represents a response to batch get connectors.
+type BatchGetConnectorsResponse = connectores.BatchGetConnectorsResponse
+
+// GetConnectorTypeSummariesRequest represents a request to get connector type summaries.
+type GetConnectorTypeSummariesRequest = connectores.GetConnectorTypeSummariesRequest
+
+// GetConnectorTypeSummariesResponse represents a response to get connector type summaries.
+type GetConnectorTypeSummariesResponse = connectores.GetConnectorTypeSummariesResponse
+
```
Comment (celaus) [inline go/connectors-client.go]: please also add the constants for endpoints, those can be used for error messages in terraform (but they only exist in Go)

Comment (celaus) [reply go/connectors-client.go]: the same goes for all clients

**Review by celaus — COMMENTED** [review]
I think the clients can be merged into a single notifications client (or notification-center-client?) with the functions merged within.

File: go/connectors-client.go (hunk 1 above)
Comment (assafad1) [reply go/connectors-client.go]: maybe I'm wrong, but I saw clients with bulk and others with batch, can you please instruct me which one to use?

File: go/connectors-client.go (hunk 2 above)
Comment (assafad1) [reply go/connectors-client.go]: added

**Review by lootag — APPROVED** [review]

**Pull request merged**